src/test/java/core/basesyntax/service/handler/impl/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service.handler.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseHandlerTest {
+    private static PurchaseHandler purchaseHandler;
+    private static FruitTransaction transaction;
+
+    @BeforeClass
+    public static void setUp() {
+        purchaseHandler = new PurchaseHandler();
+        transaction = new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple", 3);
+        Storage.fruits.put("apple", 23);
+
+    }
+
+    @Test
+    public void processOperation_Ok() {
```

Comment:
Maybe you should test the situation where someone wants to buy more fruits, than there are in the storage